src/main.js

```diff
@@ -30,4 +35,7 @@ app.use(i18n);
 // Add Cuttle window object
 initCuttleGlobals(app);
 
+// Add unHead to vue
+app.mixin(VueHeadMixin);
```

Comment:
We are slowly switching from the options API to composition, so instead of using this mixin, let's  use the composable version. Here are the [docs](https://unhead.unjs.io/setup/vue/installation#demos).

Basically we want to have the signup page call `useHead()` instead of this.head. That require s putting the useHead() call inside the setup() block of LoginView

Reply:
I agree with this 👍🏼

Reply:
I have switched unhead from using the options api to composable. Moving the useHead() call into the setup() and using a computed getter to get it to work.

[Here](https://github.com/cuttle-cards/cuttle/pull/1145/commits/7edb50266b01fdc6fa4b56043691c24c99d8fb5a)